Source/WebCore/style/PseudoElementIdentifier.h

```diff
@@ -37,4 +37,9 @@ struct PseudoElementIdentifier {
     AtomString pseudoElementNameArgument { nullAtom() };
 };
 
+inline static bool isStyleCacheable(const PseudoElementIdentifier& identifier)
```

Comment:
Not sure this helper is helpful. Cacheable where? Also it does almost nothing.

Reply:
Removed.